codecov.yml

```diff
@@ -1,18 +0,0 @@
-# Documentation: https://docs.codecov.io/docs/codecov-yaml
```

Comment:
I think we want to keep this configuration file

Reply:
Thanks a lot! This is deleted wrongly after I check the code coverage, I wanted to delete cover.html.

Reply:
I thought that might be it, thanks